<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PayStarWarsClass</class>
 <widget class="QMainWindow" name="PayStarWarsClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>763</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PayStarWars</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="verticalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>100</y>
      <width>160</width>
      <height>191</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="ButtonLayout">
     <item>
      <widget class="QPushButton" name="newGameButton">
       <property name="text">
        <string>New Game</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="loadGameButton">
       <property name="text">
        <string>Load Game</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="exitButton">
       <property name="text">
        <string>Exit</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="backGround">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>751</width>
      <height>451</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <zorder>backGround</zorder>
   <zorder>verticalLayoutWidget_2</zorder>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>763</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="PayStarWars.qrc"/>
 </resources>
 <connections/>
</ui>
